db/migrate/20160219143626_create_choices.rb

```diff
@@ -0,0 +1,10 @@
+class CreateChoices < ActiveRecord::Migration
+  def change
+    create_table :choices do |t|
+      t.string :choice_text, null: false
+      t.references :question, null: false
+      t.references :survey, null: false
```

Comment:
Do you need this foreign key if you can get the survey from this choice's parent's parent? `choice.question.survey`